src/foam/u2/crunch/CapabilityFeatureView.js

```diff
@@ -168,6 +169,21 @@ foam.CLASS({
           }
         });
       }
+    },
+    {
+      name: 'statusUpdate',
+      code: function() {
+        if ( this.cjStatus != this.CapabilityJunctionStatus.PENDING ) {
+          return;
+        };
+        this.crunchService.getJunction(null, this.data.id).then(ucj => {
+          if ( ucj && ucj.status === this.CapabilityJunctionStatus.GRANTED ) {
+            this.cjStatus = this.CapabilityJunctionStatus.GRANTED;
+          } else {
+            window.setTimeout(this.statusUpdate, 2000);
```

Comment:
Better to import: 'window', then use this.window. This will allow this code to still work if the app has multiple windows open (like the translation console does).